draft-ietf-quic-version-negotiation.md

```diff
@@ -229,8 +232,12 @@ that follows.
 The server starts incompatible version negotiation by sending a Version
 Negotiation packet. This packet SHALL include each entry from the server's set
 of Offered Versions (see {{server-fleet}}) in a Supported Version field. The
-server MAY add reserved versions (as defined in the Versions section of
-{{QUIC}}) in Supported Version fields.
+server MAY add reserved versions (as defined in {{Section 6.3 of QUIC}}) in
+Supported Version fields.
+
+A client MUST ignore a Version Negotiation packet if it contains the original
```

Comment:
I'm not a huge fan of repeating normative statements from a normative reference. How about something non-normative that's backed up by the reference:
```suggestion
Clients will ignore a Version Negotiation packet if it contains the original
```